README.md

```diff
@@ -79,6 +79,25 @@ Output artifacts:
 
 You can then use `insertion-tree.qza` for all downstream analyses, e.g. "Alpha and beta diversity analysis", instead of `rooted-tree.qza`.
 
+### Expected Runtimes
+
+We ran ``qiime fragment-insertion sepp`` on four of 32 cores of a Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz server with 265 GB available RAM. Reported "time" is accumulated "user time". Runtime only depends on number of sOTUs, not on number of samples.
+
+| QIITA ID |sOTUs	|samples|sOTU length| memory (max RSS)|time (hh:mm)|wall time (hh:mm)|
+|:--------:|-------:|------:|----------:|----------------:|:----------:|:---------------:|
```

Comment:
Can we sort via something? Perhaps sOTUS? Also, can you add links to Qiita and mention that you have to be log in to access them ?

Reply:
sorted by sOTUs, which translates to problem size